scala-thoughts.md

```diff
@@ -0,0 +1,45 @@
+# What's easy to do in Scala? What's not?
+It's really easy to do method chaining in Scala. When I needed map, then filter called on only part of a list
+I could make all of those method calls on one line and Scala was very okay with it. Additionally, objects 
+are very easy to implement in Scala because everything has to be a class already, so making the class some sort
+of intersting object is not difficult. With that, inheritence seems to be pretty easy. We only had to use Any for
+this assignment. However, from reading the documentation on Sequences and other Collections, it seemed that 
+inheritence was very common and calling super methods was simple.
+
+In Scala, it is difficult to learn collection syntax. Although it is simple to use them functionally, with map and
+filter, it was very difficult for me to learn the difference between `::`, `+:`, and `++` and some other minor syntax
+issues with collections. There were a couple other obscure syntax issues, especially with sumOfMany, which used the *
+operator to note that there were many lists. It proved very difficult to find documentation or help on some more
+obscure issues of Scala.
+
+# What is/are your favorite language design choice(s) that the designers of Scala made? Why?
+My favorite language design choice is the choice to have functional, object oriented programming. I think that they
+work really well together. Because of the method chaining that object oriented programming allows, I find it very
+readable and easy to parse because everything is in order. I do recognize, however, that this may be my bias from
+having a background with programming because a single line with a lot of methods may be more intimidating for a novice

```

Comment:
I think it may be intimidating as well, but as you point out, chaining allows the order to be more inline with the way we read and process text, so in that regard it is helpful.

I imagine mixing chaining and composition would create a confusing mess, however.